### PR TITLE
CRM-18492 Creating a report from 'all reports' put the creator's name…

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
… into the title

### CODE REVIEW

File: CRM/Core/Session.php [hunk 1]
```diff
@@ -547,6 +547,21 @@ public static function getLoggedInContactID() {
   }
 
   /**
+   * Get display name of the logged in user.
+   *
+   * @return string
+   *
+   * @throws CiviCRM_API3_Exception
+   */
+  public function getLoggedInContactDisplayName() {
```
Comment (omarabuhussein) [inline CRM/Core/Session.php]: changing the method name to **getLoggedInUserDisplayName()**   will make it more expressive  , or maybe just **getUserDisplayName()** should be enough.

**Comment by omarabuhussein** [comment]
The code looks fine but here is one concern , What if a specific user copied a report twice or more ... Then all the copied reports will have the same title .. I suggest to add the current Date-Time after the user displayName .. or maybe the Date-Time alone is enough without the (created by ... ) part . What do you think ?

**Comment by eileenmcnaughton** [comment]
I definitely think having the user name is useful - helps people figure out who to ask to delete them - but yeah, I could go with adding the date-time too.

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
@omarabuhussein I just made the change you suggested. When I tried it out in the UI I could see if was really much better & will be more useful that way.

**Comment by omarabuhussein** [comment]
Cool , I've tested this too by copying some reports on the UI side and it works perfectly. Thanks @eileenmcnaughton  Please merge.

**Comment by eileenmcnaughton** [comment]
Done!

**Pull request merged**